table.go

```diff
@@ -231,11 +238,36 @@ func (t *table) Print() {
 	t.calculateWidths()
 
 	t.printHeader(format)
+	if t.HeaderSeparatorRune != 0 {
+		t.printHeaderSeparator(format)
+	}
 	for _, row := range t.rows {
 		t.printRow(format, row)
 	}
 }
 
+func (t *table) printHeaderSeparator(format string) {
+	separators := make([]string, len(t.header))
+	separatorRuneWidth := t.Width(string([]rune{t.HeaderSeparatorRune}))
+	for index, headerName := range t.header {
+		headerRuneCount := t.Width(headerName)
+		separatorLen := headerRuneCount / separatorRuneWidth
```

Comment:
If `headerRuneCount` and `separatorRuneWidth` aren't cleanly divisible then the length here will be truncated, resulting in too short of a separator line.

Reply:
It's not a bug, it's a feautre 😄 

The intent is, that in a case where you have a separator rune that's of cell width `2` and a header of length `5`, you get a separator of length `2` (cell width `4`).

In this case I wanted the separator to be one char shorter, instead of too long. I thought better too short than too long 🤔 Either way, this is more of an edge case, I doubt many people are gonna use chars other than `_-.`.